<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEditQuery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter your Query</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="notif">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>300</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> font: rgb(255, 0, 0) 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextShowResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>661</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCutoff">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter cutOff value</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLength">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>402</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
